reference/shipping.v2.yml

```diff
@@ -1452,6 +1469,12 @@ paths:
                     description: Whether or not this shipping zone is the fallback if all others are not valid for the order.
                     example: false
                     type: boolean
+                  channels:
```

Comment:
Q: is this field only present when the merchant has more than one storefront channel?

Reply:
Like I mentioned above, this field is always present regardless of number of store channels. It's only absent in case of experiment off which would be irrelevant in this case, since this doc will be updated only once MSF is live.🍻